manifest.toml

```diff
@@ -5,7 +5,7 @@ name = "Kresus"
 description.en = "Personal finance manager"
 description.fr = "Outil personnel de gestion de finances"
 
-version = "0.20.1~ynh2"
+version = "0.20.1~ynh3"
```

Comment:
That would mark it as an update to the package, whereas nothing is new (to the user: no new features, no security fix, etc.). Can we keep the current version?

No need for users to upgrade something that is only updated on the tooling?

Reply:
Ah, it's just a rebase issue I think. Please ignore this comment in this case…

Reply:
yes, the testing and master branches are already on 0.20.1~ynh3, so not a real version change